packages/client/src/routes.ts

```diff
@@ -24,19 +31,18 @@ export const geturl = (route: string, params?: { slug: string }) => {
   return reverse(route, params);
 };
 
-type RoutePropExtended = RouteProps & { name: string };
+export const apiTopicSlugs = apiTopics.flatMap((apiTopic) =>
+  apiTopic.map((t) => t.slug)
+);
 
-export const imtrSlugs = topicsJson
-  .flatMap((api) => api.map((t) => t.slug))
-  .join("|");
+export const imtrSlugs = apiTopicSlugs.join("|");
 
-export const oloSlugs = topics
-  .filter(({ redirectToOlo, hasIMTR }) => !redirectToOlo && !hasIMTR)
+export const oloSlugs = findAllTopicsByType(TopicType.PERMIT_CHECK)
+  .filter((t) => !apiTopicSlugs.find((apiSlug) => apiSlug === t.slug))
```

Comment:
🤔

Reply:
okay... i'll add a comment

Reply:
`// Get all OLO permit checks by fetching all `PERMIT_CHECK` and filter to see if they have an actual api topic file`

Reply:
For some reason this file doesn't allow imports from utils. I would've like to use the `hasIMTR` but it didn't work. I got errors like "function does not exist" or something..

Reply:
Updated this function and closing this issue